core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.html

```diff
@@ -121,61 +144,141 @@ <h4 class="col oppia-translation-modal-author-header">
   </div>
   <div [hidden]="reviewable || !reviewMessage"
        class="oppia-suggestion-review-message">
-    <strong>Review message: </strong>{{ reviewMessage }}
+    <strong>Review message: </strong>
+    <div class="oppia-review-message">
+      {{ reviewMessage }}
+    </div>
   </div>
 </div>
 
 <div class="modal-footer">
-  <button class="btn btn-secondary" (click)="cancel()">Close</button>
-  <span ngbTooltip="Enter review message before rejecting"
-        placement="bottom"
-        [disableTooltip]="!(reviewMessage.length == 0)">
-    <button class="btn btn-danger oppia-suggestion-resolve-button"
-            *ngIf="reviewable"
-            [disabled]="reviewMessage.length == 0 || reviewMessage.length > MAX_REVIEW_MESSAGE_LENGTH || resolvingSuggestion"
-            (click)="rejectAndReviewNext()">
-      <i [hidden]="!resolvingSuggestion" class="fa fa-spinner"></i>
-      <span [hidden]="resolvingSuggestion">
-        {{ lastSuggestionToReview ? 'Reject' : 'Reject and review next' }}
-      </span>
+  <div class="oppia-review-buttons">
+    <span ngbTooltip="Enter review message before rejecting"
+          placement="bottom"
+          [disableTooltip]="!(reviewMessage.length == 0)">
```

Comment:
Can't we use the `!==` operator here instead?

Reply:
Done